data/tools/wmlxgettext

```diff
@@ -246,7 +243,7 @@ def main():
     folder = None
     filename = None
     outfile = sys.stdout
-    now = datetime.utcnow()
+    now = datetime.datetime.now(datetime.UTC)
```

Comment:
The documentation says that `datetime.UTC` has been added in version 3.11. I don't know if anyone is currently running wmlxgettext on Python 3.10, but in this case it would break the script.
Since it's just an alias for it, I suggest using `datetime.timezone.utc` instead.

Reply:
Oops, didn't realise that.